shell/imports/client/apps/applist-client.js

```diff
@@ -173,7 +172,7 @@ Template.sandstormAppListPage.helpers({
   },
 
   appMarketUrl: function () {
-    const appMarket = globalDb.collections.settings.findOne({ _id: "appMarketUrl" });
+    const appMarket = SandstormDb.collections.settings.findOne({ _id: "appMarketUrl" });
```

Comment:
This is not correct; SandstormDb is a class, and `collections` is only assigned in its constructor, so this is going to throw an exception. `globalDb` is an *instance* of that class.

The right refactor is to do this the way you did it in `appMarketHost()`; use a db instance available via the data context.

Also, creating separate prs for logically separate changes in the future would make reviewing them easier.

Thanks for taking the initiative to clean up some of this.

Reply:
I have fixed the issue you highlighted.  Thanks for the information.

As to multiple prs, that makes a lot of sense, and something I work for on my own checkins.  And sometimes my ADD tendencies get in the way, I get excited and just check it all in.